src/Orc.CsvHelper/Orc.CsvHelper.Shared/CodeGeneration/CodeGeneration.cs

```diff
@@ -95,9 +95,9 @@ private static void CreateCSharpCsvMapFile(string namespaceName, string classNam
             sb.AppendLine("{");
             sb.AppendLine(string.Format("{0}using CsvHelper.Configuration;", Spaces(4)));
             sb.AppendLine("");
-            sb.AppendLine(string.Format("{0}public class {1} : CsvClassMap<{2}>", Spaces(4), classNameMap, className));
+            sb.AppendLine(string.Format("{0}public sealed class {1} : CsvClassMap<{2}>", Spaces(4), classNameMap, className));
```

Comment:
You are allowed (actually it's preferred now) to use the new string interpolation feature:

```
$"{Spaces(4)}public sealed class {classNameMap} : CsvClassMap<{className}>"
```